bin/job

```diff
@@ -61,14 +62,15 @@ transformVM() {
   local DOMXML=$(ls $DSTD/*.xml)
 
   # we support only one disk, it needs to be aligned with DNS-1123
-  export PVCNAME=$(echo $SRC-disk-1 | sed -r 's/[_.]+/-/g')
-  export VMNAME=$(echo $SRC | sed -r 's/[_.]+/-/g')
+  export vm_name=$(echo $SRC | sed -r 's/[_.]+/-/g')
```

Comment:
Looks good but why to change the order

Reply:
Because the pvc_name depends on the vm_name so we have to calculate that first.